### ISSUE TITLE
Suggestion: Wiki

### ISSUE THREAD

**Issue by sperrye** [tanyagupta/gwg-women-techmakers#3]
Hello,
I'd like to suggest building out the repo's Wiki with information about the app's status as it grows and instructions for getting started. Thoughts?

**Comment by khusbuchandra** [comment]
Yes, definitely Wiki would be productive. It does gets easier to start for beginners like me with detailed instruction and would help hopeful contributors to get up to speed fast.

**Comment by tanyagupta** [comment]
Agreed, just assigned this to you.
